.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Documentation
+
+on: [push]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
```

Comment:
Split into several steps.
Take a look at 
https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/.github/workflows/CI.yml

Also see some parts of 
https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/trigger-rmg-tests.sh
for example the 
```
git config --global user.name "RMG Bot"
git config --global user.email "rmg_dev@mit.edu"
```